client/src/app/admin-dashboard/admin-dashboard.component.ts

```diff
@@ -31,15 +34,24 @@ export class AdminDashboardComponent implements OnInit {
       adminRegistrationFee: new FormControl(250, this.validatorsNum),
     });
 
-    this.adminEuriborDate = JSON.parse(
-      String(localStorage.getItem('adminFormData'))
-    ).adminEuriborDate;
+    this.adminService.getData().subscribe((data) => {
+      this.adminForm.setValue({
```

Comment:
If all the field names matches, you can just write this.adminForm.setValue(data)